### PR TITLE
Rewrite

### PR DESCRIPTION
**Pull request by Alkali-Metal**
To make the code-base easier to understand.

Please try to keep line lengths under 80 characters, it greatly improves readability not having to scroll sideways as well as up/down.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -6,20 +6,62 @@
...
         teams = json.load(f)
+    with open('config.json') as f:
+        configjson = json.load(f)
+    with open('whitelist.json') as f:
+        whitelistjson = json.load(f)
+
+    roles=[discord.utils.get(message.server.roles, name=configjson["modRole"]),
+    discord.utils.get(message.server.roles, name=configjson["adminRole"])]
+#               Message Prefix ----v
+    if message.content.startswith('>'):
```
Comment (MrLotU) [inline main.py]: What even is this prefix....

File: main.py [hunk 2]
```diff
@@ -114,12 +156,9 @@
...
-
-
-
-
+"""
 
 #    DO NOT LEAVE THE TOKEN IN HERE FROM NOW ON
 #    (Bad things could happen if it is public)
+
 client.run('TOKEN')
```
Comment (MrLotU) [inline main.py]: You even added that comment yourself....

File: main.py [hunk 3]
```diff
@@ -6,20 +6,62 @@
...
+        whitelistjson = json.load(f)
+
+    roles=[discord.utils.get(message.server.roles, name=configjson["modRole"]),
+    discord.utils.get(message.server.roles, name=configjson["adminRole"])]
+#               Message Prefix ----v
+    if message.content.startswith('>'):
+        command, *args = message.content[1:].split()
+        cmd = command.lower()
+        if cmd in commands:
+            response = commands[cmd](client, message, *args)
```
Comment (MrLotU) [inline main.py]: I think we miss some commands here?

**Comment by MrLotU** [comment]
@Alkali-Metal Please read the stuff I requested!

File: main.py (hunk 3 above)
Comment (MrLotU) [inline main.py]: This is not an issue yet, but fix all before opening pull requ into master

File: main.py (hunk 2 above)
Comment (MrLotU) [inline main.py]: Also not really an issue....

File: main.py (hunk 1 above)
Comment (Alkali-Metal) [inline main.py]: I use different prefixs for different things:

`!` = ready to deply / stable
`~` = in testing
`>` = in development
`any other symbol` = I'm just trying to annoy people

File: main.py (hunk 3 above)
Comment (Alkali-Metal) [inline main.py]: Nope, nothing is missing there afaik

File: main.py (hunk 2 above)
Comment (Alkali-Metal) [inline main.py]: Shhh, I'm a :fish:

**Pull request merged**